tests/composer/test_blueprints.py

```diff
@@ -121,6 +121,15 @@
 
 
 class BlueprintsTest(unittest.TestCase):
+    @unittest.skipUnless(os.path.exists("/run/weldr/api.socket"), "Setup requires a running API server")
+    @classmethod
+    def setUpClass(cls):
+        """
+            Make sure that example blueprints are pushed to the server!
+        """
+        rc = blueprints_push("/run/weldr/api.socket", 0, ["example-http-server.toml"], show_json=False)
```

Comment:
It will also need example-development

Reply:
I had the impression example-development.toml is uploaded by one of the tests but I will add it.

Reply:
It depends on which test, when I run the nose tests against osbuild-composer it complains that example-devel isn't installed.